modules/frontend/search_handlers.go

```diff
@@ -50,8 +50,8 @@ func newSearchStreamingGRPCHandler(cfg Config, next pipeline.AsyncRoundTripper[c
 		}
 
 		var finalResponse *tempopb.SearchResponse
-		c := combiner.NewTypedSearch(int(limit))
-		collector := pipeline.NewGRPCCollector[*tempopb.SearchResponse](next, cfg.ResponseConsumers, c, func(sr *tempopb.SearchResponse) error {
+		comb := combiner.NewTypedSearch(int(limit))
```

Comment:
this is just cleanup. IDE was complaining about shadowing the package name combiner